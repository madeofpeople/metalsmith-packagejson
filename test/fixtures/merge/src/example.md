---
template: layout.html
title: Hello World!
---
This is an example.