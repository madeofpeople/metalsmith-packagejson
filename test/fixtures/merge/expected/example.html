<html>
<body>
metalsmith-packagejson-merge
MIT
git
git+https://github.com/madeofpeople/metalsmith-packagejson.git

<p>This is an example.</p>

</body>
</html>
